tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -427,6 +468,12 @@ def _create_bucket(spinner, cloud_logger, config, bucket_name,
         _write_spinner_and_log(
             spinner, cloud_logger,
             'ACLs successfully copied over from the source bucket')
+    #handle Uniform bucket access buckets as every Bucket creation is Fine grained. So enable UniformBucketLevelAccess if ACL does not exist
+    else:
```

Comment:
Current design is to replicate all source bucket permissions to target bucket.  Instead of else this functionality can be   achieved by assigning the source_bucket.iamconfiguration to iamconfiguration of target_bucket at the beginning of current method similar to other properties such as storage_class, requester_pays etc